string objectpoint_name_destination
---
string objectpoint_name